tests/wps_restapi/test_jobs.py

```diff
@@ -1298,7 +1305,7 @@ def test_get_job_invalid_uuid(self):
         """
         # to make sure UUID is applied, use the "same format" (8-4-4-4-12), but with invalid definitions
         base_path = sd.job_service.path.format(job_id="thisisnt-some-real-uuid-allerrordata")
-        for sub_path in ["", "/inputs", "/outputs", "/results", "/logs", "exceptions"]:
+        for sub_path in ["", "/inputs", "/outputs", "/results", "/logs", "exceptions", "transforms"]:
```

Comment:
I think the `/` was missing from `"exceptions"` by mistake and should also be provided in `"/transforms"`.